hack/release.sh

```diff
@@ -25,6 +25,7 @@ readonly SERVING_RELEASE_GCR
 # istio.yaml file to upload
 # We publish our own istio.yaml, so users don't need to use helm"
```

Comment:
While you're here, do you mind removing the stray " at the end of this line? Thanks! :)

Reply:
Done